pyproject.toml

```diff
@@ -0,0 +1,3 @@
+[build-system]
+requires = ["setuptools>=40.8.0", 'NEURON >=7.7']
+build-backend = "setuptools.build_meta:__legacy__"
```

Comment:
To be honest, I don't understand the implications of the version requirement (Feb 2019) nor the `__legacy__` flag. I guess it's a safe bet that our pip-based users will be fine with both.